rosidl_typesupport_introspection_cpp/QUALITY_DECLARATION.md

```diff
@@ -105,15 +118,18 @@ This includes:
 - achieving and maintaining a reasonable branch line coverage (90-100%)
 - no lines are manually skipped in coverage calculations
 
-Current coverage statistics can be viewed [here](https://ci.ros2.org/job/nightly_linux_coverage/lastSuccessfulBuild/cobertura/). A description of how coverage statistics are calculated is summarized in this page ["ROS 2 Onboarding Guide"](https://docs.ros.org/en/rolling/Contributing/Developer-Guide.html#note-on-coverage-runs).
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/nightly_linux_coverage/lastSuccessfulBuild/cobertura/). Given the different approach used for testing (detailed in "Note about the design of testing" in this same document) the pattern to search for `rosidl_typesupport_introspection_cpp` is:
```

Comment:
@j-rivero 
```suggestion
Current coverage statistics can be viewed [here](https://ci.ros2.org/job/nightly_linux_coverage/lastSuccessfulBuild/cobertura/). Given the approach used for testing (detailed in "Note about tests design" at the beginning of section [4]) the pattern to search for `rosidl_typesupport_introspection_cpp` is:
```
I missed this one in #656.

Reply:
I'm going to include it in this PR, not a big deal anyway. 493e21e